docs/data-scientists/experiment-sizing.md

```diff
@@ -0,0 +1,65 @@
+---
+id: experiment-sizing
+title: Sizing Experiments Using Mozanalysis
+slug: /experiment-sizing
+---
+
+This page gives an overview of how Mozanalysis can be used to do experiment sizing. **[Mozanalysis]** is a library used to standardize experiment analysis at Mozilla. Mozanalysis contains the statistical tools used by [Jetstream] to produce results from experiments, as well as tools to pull historical data from BigQuery to calculated necessary sample sizes to carry out experiments.
+
+## Experiment sizing paradigm
+
+Experiment sizing in Mozanalysis retrieves historical data from BigQuery in a way that mimicks how experiment enrollment and analysis is conducted by [Jetstream]: an enrollment period is defined, during which clients that satisfy certain target conditions are recorded; following enrollment, metrics are calculated for those clients over a defined analysis period. Following collection of metrics, sample size calculation is performed that corresponds to the statistical tests to be used in analyzing the eventual experiment. 
+
+<img src="/img/mozanalysis/mozanalysis-sizing.png" alt="Mozanalysis sizing workflow" className="img-lg"/>
+
+### Enrollment and analysis
+
+Similar to how experiments use an enrollment period to select clients for a study before collecting metrics for a period of time, Mozanalysis experiment sizing records all clients that satisfy the target conditions during an enrollment period, set by the user based on the analysis start date and the number of dates to enroll. Metrics are then recorded for a user-defined number of days for the analysis.
+
+<img src="/img/mozanalysis/enrollment-period.png" alt="Analysis with an enrollment period" className="img-lg"/>
```

Comment:
I don't think this visual corresponds to what your code is actually doing. For a given client, observation begins immediately after enrollment, as the first analysis windows (for the daily, weekly, monthly levels) will begin at zero days after enrollment. 

 Check out [this image](https://experimenter.info/configuring#visual-explanation) as an example. Basically, the yellow arrows should be shifted back to start at the blue enrollment rectangles.

Reply:
That's my bad, these images were created when sizing _did_ work this way, and I failed to update the images (or proofread)